google-cloud-bigtable/pom.xml

```diff
@@ -104,6 +104,32 @@
         <javadoc.opts>-Xdoclint:none</javadoc.opts>
       </properties>
     </profile>
+
+    <profile>
+      <id>bigtable-dev</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>com.theoryinpractise</groupId>
+            <artifactId>googleformatter-maven-plugin</artifactId>
```

Comment:
I would prefer to apply this to all of google-cloud-java instead of just google-cloud-bigtable

Reply:
Extracted to https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2851